tools/dashboard/src/app/dashboard-app.ts

```diff
@@ -1,8 +1,32 @@
 import {Component} from '@angular/core';
+import {AngularFireDatabase, FirebaseListObservable} from 'angularfire2/database';
 
 @Component({
   selector: 'dashboard-app',
   templateUrl: './dashboard-app.html',
   styleUrls: ['./dashboard-app.css']
 })
-export class DashboardApp {}
+export class DashboardApp {
+
+  /** Observable that emits all payload results from Firebase. */
+  payloads: FirebaseListObservable<PayloadResult[]>;
+
+  constructor(database: AngularFireDatabase) {
+    this.payloads = database.list(`payloads`);
+  }
+}
+
+/** Interface that describes the payload results from the Firebase database. */
+interface PayloadResult {
```

Comment:
Thought about it but wanted to wait until we have more other components. I'm changing it anyway now.